app/views/users/_show_user_info.html.haml

```diff
@@ -11,3 +11,33 @@
 - if can? :show_full, @user
   %p #{@user.point} #{point_image}
   /= link_to 'Pasar', market_items_path, class: 'btn'
+
+-if can? :edit, @user.about_user
+  %h2 About Me
+  -if @user.about_user.nil?
+    %p Buat sekarang!
+    = form_for [@user, @user.build_about_user], url: user_about_user_path(@user) do |f|
```

Comment:
Can't you use the `_form` partial here?

Reply:
Seems like this cannot be done due to `_form` partial which takes in `@user.about_user` instead of `@user.build_about_user`

Reply:
Can look into https://guides.rubyonrails.org/v4.2/layouts_and_rendering.html#using-partials section 3.4.4? Seems like we can pass in variables to the partials.

Can you check if this has been used in some of the other partials we have used? I think we pass local variables to the partials in some way, though I can't really remember how.

Reply:
Yup, seems like passing in local variables inside partials is the way 🥳 !